functions/fisher.fish

```diff
@@ -82,7 +82,7 @@ function fisher --argument-names cmd --description "A plugin manager for Fish"
 
                 command mkdir -p $source/{completions,conf.d,themes,functions}
 
-                $fish_path --command "
+                $fish_path --no-config --command "
```

Comment:
What does --no-config do?

Reply:
As stated in #748 it avoids recursion, making easier to write confs files (not having to test for `status is-interactive`)
I've checked the command after line 85 and it seems to work fine with fish without config, as it should not require any user customizations
In addition, not reading config when it's not necessary should result in faster execution.
This `--no-config` flag was added on https://github.com/fish-shell/fish-shell/pull/7921 in order to fix https://github.com/fish-shell/fish-shell/issues/5394, that is, precisely avoid guarding with `status --is-interactive`

Reply:
> faster execution

Awesome. 💯 

If we start using --no-config (and I think we should) we don't need to add this to the docs.

Reply:
@Susensio Could you open a new PR that only includes the --no-config fix? Or modify this one like so. You are welcome to open a new PR to improve the docs if you want.